foundry.toml

```diff
@@ -5,7 +5,7 @@
   ]
```

Comment:
*Arbitrary comment location*

Why did the audit report markdown file change? Maybe your editor auto formatted it? Let's revert that change.

Reply:
Yeah.. must have gotten pulled on a global search, then accidentally saved/formatted.. fix: https://github.com/withtally/staker/pull/121/commits/52a3e588ed778cdcb420e73ea5732c9a03a67748